alvr/server_core/src/web_server.rs

```diff
@@ -61,14 +62,55 @@ pub async fn web_server(connection_context: Arc<ConnectionContext>) -> Result<()
     }
 
     let router = Router::new()
-        .route("/api/dashboard-request", routing::post(dashboard_request))
-        .route("/api/events", routing::get(events))
-        .route("/api/set-buttons", routing::post(set_buttons))
-        .route(
-            "/api/version",
-            routing::get(async || alvr_common::ALVR_VERSION.to_string()),
+        .nest(
+            "/api",
+            Router::new()
+                .route("/events", routing::get(events_websocket))
+                .route("/log", routing::post(set_log))
+                .nest(
+                    "/session",
+                    Router::new()
+                        .route("/", routing::get(get_session).post(update_session))
+                        .route("/values", routing::post(set_session_values))
+                        .route(
+                            "/client-connections",
+                            routing::post(update_client_connections),
+                        ),
+                )
+                .route("/buttons", routing::post(set_buttons))
+                .route("/insert-idr", routing::post(insert_idr))
+                .route("/capture-frame", routing::post(capture_frame))
+                .nest(
+                    "/recording",
+                    Router::new()
+                        .route("/start", routing::post(start_recording))
+                        .route("/stop", routing::post(stop_recording)),
+                )
```

Comment:
Might be a bit cleaner to put this into a `debug` namespace (for which we could maybe have reduced stability...).